project/Dependencies.scala

```diff
@@ -75,11 +75,10 @@ object Dependencies
     val driverCore = "com.datastax.oss" % "java-driver-core-shaded" % DataStaxJavaDriver driverCoreExclude()
```

Comment:
theoretically, because everything will be in assembly, we won't need to exclude `ffi` dependencies.

Reply:
I would like to keep things consistent between the main artifact and the shaded one. Do you think it would be beneficial to have ffi in the shaded version?

Reply:
My thought is following - we have problem with ffi when we're using connector with Spark directly via `--packages`. If instead we'll use shaded version with `--packages`, and leave the `ffi` dependencies for connector that will be used for building applications, then we won't have problem with Spark.